assets/src/dashboard/components/typeahead-input/index.js

```diff
@@ -71,14 +76,30 @@ InputContainer.propTypes = {
   isOpen: PropTypes.bool,
 };
 
+const ControlVisibilityContainer = styled.div`
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-start;
+
+  @media ${({ theme }) => theme.breakpoint.mobile} {
+    visibility: ${({ isExpanded }) => (isExpanded ? 'visible' : 'hidden')};
```

Comment:
Is toggling `visibility` needed?  Wouldn't opacity being set to 0 and 1 be equivalent? Just curious.

Reply:
well now that you mention it :) it's not needed and actually fixes the issue i was running into with an input ref to focus on click - so thank you for pointing this out. I'd started with visibility and later swapped to using opacity for the transition because it's smoother and just didn't think about how it made visibility redundant.